packages/suite/src/actions/suite/bioAuthThunks.ts

```diff
@@ -1,84 +1,27 @@
-import { createSingleInstanceThunk, createThunk } from '@suite-common/redux-utils';
+import { createThunk } from '@suite-common/redux-utils';
 import { notificationsActions } from '@suite-common/toast-notifications';
-import { isMacOs, isWeb } from '@trezor/env-utils';
 import { desktopApi } from '@trezor/suite-desktop-api';
 
-import { TranslationFunction } from 'src/hooks/suite/useTranslation';
-import {
-    BLUR_LOCK_TIMEOUT_MS,
-    selectBioAuthEnabled,
-    selectBlurTimeoutId,
-    selectIsBioAuthAvailableStateKnown,
-    selectIsBioAuthValidationRequested,
-    selectIsBioAuthValidationRequired,
-    selectIsRequestingBioAuthChange,
-} from 'src/reducers/bioAuth';
 import { Dispatch } from 'src/types/suite';
 
 import { bioAuthActions } from './bioAuthActions';
 import * as storageActions from './storageActions';
 
 const BIO_AUTH_PREFIX = '@suite/bioAuth';
 
-export const bioAuthWindowBlurThunk = createThunk(
-    `${BIO_AUTH_PREFIX}/bioAuthWindowBlurThunk`,
-    (date: Date, { dispatch, getState }) => {
-        const blurTimeoutId = selectBlurTimeoutId(getState());
-        if (blurTimeoutId) {
-            clearTimeout(blurTimeoutId);
-        }
-
-        const timeoutId = setTimeout(() => {
```

Comment:
This was important: You shouldn't just rely on time, because JS time can be changed in the OS settings.
Hence added a timeout, that ensures that the app is locked after the elapsed time. So you couldn't bypass it by changing the OS time.